analyzers/packaging/SonarAnalyzer.CSharp.nuspec

```
@@ -16,7 +16,7 @@
     <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.52.0.0</releaseNotes>
     <language>en-US</language>
     <copyright>Copyright © 2015-2022 SonarSource SA</copyright>
-    <tags>Roslyn Analyzers Refactoring CodeAnalysis</tags>
+    <tags>Roslyn Analyzers Refactoring CodeAnalysis CleanCode</tags>
```

What about this?
```suggestion
    <tags>Roslyn Analyzers Refactoring CodeAnalysis CleanCode Clean Code</tags>
```
There are only 8 packages that use "CleanCode" directly: https://www.nuget.org/packages?q=Tags%3A%22CleanCode%22

There are 168 packages that use "clean" (code, architecture, log, design, ....)
https://www.nuget.org/packages?q=Tags%3A%22Clean%22

I don't know how to search for "Clean" && "Code", unfortunately.

Maybe marketing has an answer to what we should use? As it's similar to SEO and they might already have a strategy about where we want to be.

---

I think there is nothing wrong with adding both "Clean" and  "Code". Will add them.